actions/earned-income-budgeting.vbs

```diff
@@ -1482,39 +2500,43 @@ For ei_panel = 0 to UBOUND(EARNED_INCOME_PANELS_ARRAY, 2)       'looping through
                             total_checks = 0
 
                             'I guess we do this again - maybe we can REMOVE CODE
-                            actual_checks_provided = FALSE
+							'TESTING TO SEE IF THIS CAN BE REMOVED
+                            ' actual_checks_provided = FALSE
                             For all_income = 0 to UBound(LIST_OF_INCOME_ARRAY, 2)
                                 If LIST_OF_INCOME_ARRAY(pay_date, all_income) = "" AND LIST_OF_INCOME_ARRAY(gross_amount, all_income) = "" AND LIST_OF_INCOME_ARRAY(hours, all_income) = "" Then LIST_OF_INCOME_ARRAY(panel_indct, all_income) = ""
-                                If LIST_OF_INCOME_ARRAY(panel_indct, all_income) = ei_panel Then
-                                    'ADD ERROR HANDLING HERE
-                                    actual_checks_provided = TRUE
-                                End If
+                            '     If LIST_OF_INCOME_ARRAY(panel_indct, all_income) = ei_panel Then
+                            '         'ADD ERROR HANDLING HERE
```

Comment:
2508: Not sure if you intended to leave this comment here or if it's a future reminder.

Reply:
Good catch, this was a weird place where I had duplicated functionality and I was messing around with it. I've cleaned up the comments and removed the unnecessary code.